src/pymovements/utils/archives.py

```diff
@@ -149,12 +150,22 @@ def _extract_tar(
         Path to the directory the file will be extracted to.
     compression: str | None
         Compression filename suffix.
+    skip: bool
+        Skip already extracted files. (default: True)
     """
     with tarfile.open(source_path, f'r:{compression[1:]}' if compression else 'r') as archive:
-        if sys.version_info < (3, 12):  # pragma: <3.12 cover
-            archive.extractall(destination_path)
-        else:  # pragma: >=3.12 cover
-            archive.extractall(destination_path, filter='tar')
+        for member in archive.getnames():
+            if (
+                    os.path.exists(os.path.join(destination_path, member)) and
+                    member[-4:] not in _ARCHIVE_EXTRACTORS and
```

Comment:
probably it would be nice to add a short note that archives are extracted regardless of the value of `skip`/ `resume`